eng/pipelines/templates/pipeline.yml

```diff
@@ -134,7 +134,7 @@ jobs:
     name: DDNuGet-Windows
     demands:
     - DotNetFramework
-    - Allow_NuGet_E2E_Tests -equals true
+    - Agent.Name -equals DDNuGetW004
```

Comment:
Before merging we should install Dev17 on all of the E2E/Apex machines and undo this demands change.

Reply:
Tracking issue https://github.com/NuGet/Client.Engineering/issues/988

Reply:
Resolved! 😄